src/templates/steps-links-view.html

```diff
@@ -29,3 +30,37 @@
     </ul>
   </div>
 {% endif %}
+
+<!-- RELATED ITEMS/EXPERIMENTS -->
+{% if Entity.type == 'items' %}
+{% if relatedItemsArr %}
+  <div class='mt-2'>
+    <h4 class='d-inline'>{{ 'Related items'|trans }}</h4>
+    <ul class='list-group'>
+{% for link in relatedItemsArr %}
+      <li class='list-group-item'>
+        <i class='fas fa-link mr-1'></i>
+{% if link.bookable %}
+        <i class='far fa-calendar-alt'></i>
+{% endif %}
+        {{ link.name|raw }} - <a href='database.php?mode=view&id={{ link.entityid }}'>{{ link.title|raw }}</a>
+      </li>
+{% endfor %}
+    </ul>
+  </div>
+{% endif %}
+
+{% if relatedExperimentsArr %}
+  <div class='mt-2'>
+    <h4 class='d-inline'>{{ 'Related experiments'|trans }}</h4>
+    <ul class='list-group'>
+{% for link in relatedExperimentsArr %}
+      <li class='list-group-item'>
+        <i class='fas fa-link mr-1'></i>
+        <a href='experiments.php?mode=view&id={{ link.entityid }}'>{{ link.title|raw }}</a>
+      </li>
+{% endfor %}
+    </ul>
+  </div>
+{% endif %}
```

Comment:
this is not properly indented, use 2 spaces for html, see .editorconfig

Reply:
Should be fixed. I used the <!-- LINKS --> section as template.